src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -87,6 +89,8 @@
 
     private DynamicSpout spout;
 
+    private PersistenceAdapter persistenceAdapter;
```

Comment:
Doc block these.  Also I like to have getters, and use em 👍

Reply:
oh you have a getter, we should update this class to use it

Reply:
Done.